src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs

```diff
@@ -74,7 +72,9 @@ internal sealed class ReflectPropertyDescriptor : PropertyDescriptor
         private MethodInfo _shouldSerializeMethod;      // the should serialize method
         private MethodInfo _resetMethod;                // the reset property method
         private EventDescriptor _realChangedEvent;           // <propertyname>Changed event handler on object
+#pragma warning disable CS0649
```

Comment:
If nothing assigns `_receiverType` on `!FEATURE_INOTIFYPROPERTYCHANGED`, could the field be removed under those builds instead of silencing a warning?